src/Identity/UI/src/wwwroot/V4/css/site.css

```diff
@@ -7,9 +7,14 @@ a.navbar-brand {
   word-break: break-all;
 }
 
-/* Provide sufficient contrast against white background */
+/* Provide sufficient contrast against white background and black text */
 a {
-  color: #0366d6;
+  color: #0077cc;
+}
+
+.form-control:focus {
```

Comment:
Note: I'm a little worried we are breaking from pattern/convention here, as prior to this site.css is identical in the template and identity UI.  @javiercn will this cause issues in scaffolding? i.e. should we push the form-control focus to site css as well in the templates?

Reply:
That's interesting. Why do we have this file if it's identical to what's in the site anyway?

Reply:
@javiercn is the source of truth, but I believe many of the things in identity UI, like its copies of bootstrap and site.css are backups/fallbacks in the situation where the wwwroot in the site does not contain these files.  But I'm just speculating from what I recall

Reply:
Ah, makes sense! In that case it does seem right to put this in the site.css in the app too then.